### PR TITLE
New branch created for suggested editing changes to Appendix B.

### PR DESCRIPTION
**Pull request by anselld**
Hi @Empiech007, please see my suggested changes to Appendix B. I see a few problems with some of the equations, e.g. Corollary 2 - the font of the text parts ('for all') is a serif font and not a sans serif font, as used for text elsewhere.
Also, there are some strange characters in the Lemma 2 equations.

### CODE REVIEW

File: src/cryptography/amortization-bp-ipp/mainreport.md [hunk 1]
```diff
@@ -985,60 +984,48 @@ The **total cost** of computing the coefficients for $n = 32$ using *Algorithm 1
...
-
-
+The proof of **Theorem 1** makes use of a few basic properties of the parity values $b(i,j)​$, which are described in 
+this appendix.
 
 ### Notation and Definitions
 
-The letters  $j$,  $k$  and  $ l$  denote non-negative integers unless otherwise stated. In addition, 
-$n = 2^k$, $ i \in \{ 0, 1, 2, ... , n-1 \}$, and  $j \in \{ 1, 2, ... , k \}$. 
+The letters $j$, $k$ nd $l$ denote non-negative integers, unless otherwise stated. In addition:
```
Comment (Empiech007) [inline src/cryptography/amortization-bp-ipp/mainreport.md:992]: Please note that `$k$ nd $l$` should be `$k$ and $l$`

Comment (anselld) [reply src/cryptography/amortization-bp-ipp/mainreport.md:992]: Sorry for that @Empiech007. Shall I make the change on my side and push the report again?

**Comment by hansieodendaal** [comment]
@Empiech007, 
- please fix the `for all` usage in all four equations of `Corollary 2` to use math notation, or provide the corrections to @anselld;
- please redo your comment on line 173 from the main branch so it can be fixed.

**Comment by anselld** [comment]
@hansieodendaal and @Empiech007 
Line 173 has been changed as requested:
Make sure the brackets and the enclosed text reads as $ [ 0 , 2^n - 1 ]$

File: src/cryptography/amortization-bp-ipp/mainreport.md [hunk 2]
```diff
@@ -170,7 +170,7 @@ of it.
 ### Recursion in Bulletproofs Inner-product Proof
 
 In Bulletproofs range proofs, a prover commits to a value $v$ and seeks to construct an IPP to the fact 
-that $v \in ( 0 , 2^n ) $. Pedersen commitments are used to keep the value of $v$ confidential, and are expressed as 
+that $v \in $ [ 0 , 2^n - 1 ]$. Pedersen commitments are used to keep the value of $v$ confidential, and are expressed as
```
Comment (Empiech007) [inline src/cryptography/amortization-bp-ipp/mainreport.md:173]: Okay, it is correct. But please remove one of the dollar signs, this one here ` \in $ [ 0`

Comment (anselld) [reply src/cryptography/amortization-bp-ipp/mainreport.md]: Thanks, @Empiech007, done. Will commit again.

**Comment by Empiech007** [comment]
> @Empiech007,
> 
> * please fix the `for all` usage in all four equations of `Corollary 2` to use math notation, or provide the corrections to @anselld;
> * please redo your comment on line 173 from the main branch so it can be fixed.

Okay, done. Thank you @anselld and @hansieodendaal

**Comment by Empiech007** [comment]
Hi @hansieodendaal. Please merge to master. I would like to concentrate on the next report.

**Pull request merged**